Classes/Mailer.php

```diff
@@ -144,7 +144,9 @@ private function setTitle($htmlSrc)
     {
         // Extract title from HTML
         preg_match('|<title[^>]*>(.*)</title>|i', $htmlSrc, $m);
-        $title = trim($m[1]);
+
+        // As this is being extracted from HTML and it is being used as an email subject we need to decode any entities.
+        $title = trim(html_entity_decode($m[1]));
```

Comment:
What is there to test???  The code is extracting the subject from HTML source (not my code) your test is making bad assumptions as you are not testing for this (this is a classic example of why I think coders should not write their own unit tests). If you are setting HTML content as a subject in an email it needs to be devoid of html_entities as it is not supported.

Covered in RFC's for mail headers:
https://tools.ietf.org/html/rfc2047
https://tools.ietf.org/html/rfc5890
https://tools.ietf.org/html/rfc6532

You are allowed to have entities in message bodies which are HTML.